src/util.js

```diff
@@ -3,8 +3,11 @@ export function contains(arrayLike, stringOrRegex) {
 		return arrayLike.indexOf(stringOrRegex) > -1;
 	}
 
-	const match = arrayLike.find(function(item) {
-		return item.match(stringOrRegex);
+	var match;
+	arrayLike.forEach(function(item) {
+		if (item.match(stringOrRegex)) {
+			return match = item.match(stringOrRegex);
```

Comment:
Functionally the same, but splitting this into two statements (the assignment and the `return`) might be clearer - might just be me, but returning an assignment tripped me up for a second! (Just not something I commonly see is all.)